buildkite.sh

```diff
@@ -0,0 +1,356 @@
+#!/bin/bash
+
+cat << 'EOF'
+steps:
+  - command: |
+      uname -a
+      make --version
+      \${SHELL} --version || true
+      c++ --version
+      ld -v
+      ! command -v gdb &>/dev/null || gdb --version
+      dmd --version || true
+    label: "Print envs"
+    env:
+      DETERMINISTIC_HINT: 1
+
+  - command: |
```

Comment:
The script is very long, might want to put it into a separate bash file or so.

Reply:
The idea of putting everything in one place was that it's easy to `wget` in other repos. Though of course we could almost as easily wget the entire zip / tar.gz snapshot of this repo.